.gitignore

```diff
@@ -38,3 +38,5 @@ CMakeFiles/
 CTestTestfile.cmake
 .idea/
 .githooks/clangFormatMac
+/cmake-build-debug/
+/build/
```

Comment:
I would like to see this as a separate commit. (Following the principle of "One Commit. One Change.") Why do those directories appear in your source tree? Are they specific to a particular tool or platform? I'm fine with the change itself, but I would like some clarification!

Reply:
Hi,  sorry for the late reply due to other obligations.
This can be safely ignored, i did not realize this change in .gitignore